plugin.js

```diff
@@ -440,6 +460,37 @@
 			return items.length;
 		};
 
+		/**
+		 * Suppresses change events until after delegateFunction has been executed.
+		 * 
+		 * Performing multiple successive updates to a PlaceholdersCollection object
+		 * can cause UI "flickering" as multiple change events occur. In some cases
+		 * it may be useful to suppress changes until after these operations have 
+		 * been executed.
+		 * 
+		 * @param {Function} delegateFunction The function that will be performing
+		 * multiple calls to this PlaceholdersCollection object. @this will be set
+		 * to the instance of PlaceholdersCollection.
+		 * @returns {number} The new length of the collection.
+		 */
+		this.batchChanges = function(delegateFunction)
```

Comment:
Let's name the argument `callback` please to keep it in line with the other callback type arguments. Also it would be good if there where a type defined, similar to `processPlaceholdersCallback`, where `@this` could be used in an inspectable fashion, like

```
/**
 * @callback batchProcessCallback
 * @this PlaceholdersCollection
 */